rust/ql/test/query-tests/security/CWE-696/test.rs

```diff
@@ -123,10 +123,10 @@ unsafe fn harmless2_11() {
 // --- transitive cases ---
 
 fn call_target3_1() {
-    _ = stderr().write_all(b"Hello, world!"); // $ MISSING: Alert=source3_1 Alert=source3_3 Alert=source3_4
+    _ = stderr().write_all(b"Hello, world!"); // $ Alert=source3_1 Alert=source3_3 MISSING: Alert=source3_4
```

Comment:
The `source3_4` result is still missing because the call to `bad3_3()` has no `getStaticTarget()`.